apps/wallet/src/ui/app/hooks/useGetInactiveValidators.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useRpcClient } from '@mysten/core';
+import { type ObjectId } from '@mysten/sui.js';
+import { useQuery } from '@tanstack/react-query';
+
+import { notEmpty } from '_helpers';
+
+// Get inActive validators by inactivePoolsIds
+export function useGetInactiveValidators(inactivePoolsId?: ObjectId) {
+    const rpc = useRpcClient();
+    const data = useQuery(
+        ['inactive-pool-id', inactivePoolsId],
+        () => rpc.getDynamicFields({ parentId: inactivePoolsId! }),
+        { enabled: !!inactivePoolsId }
+    );
+    // return validator stakePoolId
+    return {
+        ...data,
+        data:
+            data.data?.data
```

Comment:
I think it would be better to transform your data using the `select` option provided by `useQuery` (https://tkdodo.eu/blog/react-query-data-transformations), then you can just return the `useQuery` result directly instead of having to override the data property

That also gets rid of the `data.data.data` stuff which is kinda confusing